test/tests/rackhd20/test_rackhd20_api_notifications.py

```diff
@@ -0,0 +1,269 @@
+'''
+Copyright 2017 Dell Inc. or its subsidiaries.  All Rights Reserved.
+Author(s):
+'''
+
```

Comment:
Please add a description as to what the test is doing here.  There isn't anything in the PR nor here to call it out.
thanks!!